tests/python/tests/run_unittests.sh.in

```diff
@@ -0,0 +1 @@
+LD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/src/: PYTHONPATH=${CMAKE_BINARY_DIR}/src/python/ WITH_LIBMODULEMD=${WITH_LIBMODULEMD} ${PYTHON_EXECUTABLE} -m unittest discover -s ${CMAKE_CURRENT_SOURCE_DIR}/../
```

Comment:
Why is there a `../` at the very end of the line? That wasn't there before... I might be missing something.

Reply:
Because unlike with python-nose, the starting directory must be the parent directory of the tests and not the tests, otherwise there are errors with relative imports.